atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -176,7 +178,7 @@ def _ensure_image_registry(self, image):
             raise RuntimeError("Shouldn't happen, images should have already "
                                "registry set in dockerfile_images")
 
-    def _pull_and_tag_image(self, image, nonce):
+    def _pull_and_tag_image(self, image: ImageName, nonce: str) -> Optional[ImageName]:
```

Comment:
I don't think the return value is Optional, the only `return` returns an ImageName and if the return doesn't happen then you get a RuntimeError